TestApp/package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "./node_modules/.bin/eslint ."
   },
   "dependencies": {
+    "@react-native-community/async-storage": "^1.5.0",
```

Comment:
@dhei Should we consider pinning this version or is it OK to allow newer ones?

Reply:
I don't have a strong preference. NPM's default behavior is to use highest minor version (no pinning) and sometimes minor versions contain breaking change therefore breaking the app. 

It's probably okay not pinning version and use `^` for external dependencies except `react` and `react-native`.

Reply:
@jwallra Btw, thanks for bringing this up. We can revisit this decision if we run into problems of not pinning version.